app/models/assignment.rb

```diff
@@ -320,7 +320,7 @@ def current_stage_name(topic_id = nil)
   # check if this assignment has multiple review phases with different review rubrics
   def varying_rubrics_by_round?
     # E-2084 corrected '>=' to '>' to fix logic
```

Comment:
Put a comment - "This is a hack, we should actually check if we have more than one rubric of a given type eg, review